www/src/components/IconsTable.jsx

```diff
@@ -122,18 +109,20 @@ function IconsTable() {
   }, []);
 
   useEffect(() => {
-    debouncedSetIconsList(
-      ICON_NAMES.filter(name => name.toLowerCase().includes(searchValue.toLowerCase())),
-    );
-  }, [searchValue, debouncedSetIconsList]);
+    const list = ICON_NAMES.filter(name => name.toLowerCase().includes(searchValue.toLowerCase()));
+    if (typeof searchValue === 'string' && searchValue.trim() !== '') {
```

Comment:
do we really need to check for `typeof searchValue === 'string'` here? I think it's always a string (even if you type a number into the search field)

Reply:
Yes, thanks